tracer/src/Datadog.Trace/AppSec/Waf/Waf.cs

```diff
@@ -115,6 +115,19 @@ public bool UpdateRules(IEnumerable<RuleData> res)
             return ret == DDWAF_RET_CODE.DDWAF_OK;
         }
 
+        public bool ToggleRules(IDictionary<string, bool> ruleStatus)
+        {
+            if (ruleStatus == null || ruleStatus.Count == 0)
+            {
+                return false;
+            }
+
+            var encoded = encoder.Encode(ruleStatus);
```

Comment:
doesn't this need to be disposed?

Reply:
Not "needed". It will be cleansed when garbage collected (Disposed in destructor). I mean, it won't hurt to call Dispose, but it is not called on any of the places where it is used.